frontend/src/components/permissions/components/edit-permission-element/edit-permission-element.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useForm } from 'react-hook-form';
+import { Checkbox } from 'components/common';
+import { IUserWithPermissions } from 'common/interfaces';
+import { PermissionsActionCreator } from 'store/slices';
+import { PermissionName } from 'common/enums';
+import { RootState } from 'common/types';
+
+type Props = {
+  user: IUserWithPermissions;
+  nameOfPermission: string;
+};
+
+const EditPermissionElement: React.FC<Props> = ({ user, nameOfPermission }) => {
+  const { permissions } = useSelector(({ permissions }: RootState) => ({
+    permissions: permissions.permissions,
+  }));
+  const dispatch = useDispatch();
+
+  const userPermissions = user.permissions!.map((permission)=>permission.name);
```

Comment:
Move it to frontend/src/components/permissions/components/edit-permission-element/helpers/
`getPermissionNames`